packages/php-db-import-export/src/Backend/Bigquery/ToStage/FromGCSCopyIntoAdapter.php

```diff
@@ -69,4 +83,16 @@ public function runCopyCommand(
 
         return $ref->getRowsCount();
     }
+
+    private function waitUntilJobIsDone(Job $job): void
+    {
+        // do linear back off 3 times
+        for ($i = 0; $i < 3; $i++) {
+            if ($job->isComplete()) {
+                break;
+            }
+            sleep(28800); //8h
+            $job->reload();
+        }
```

Comment:
Eh to asi ne :D to to bude čekat 3x po 8mi hodinách :D

Reply:
bych řekl, že ta suita doběhla protože je ten job hned complete

Reply:
Odobral som to pouzijeme ten defaultny backoff co uz je v tej metode `$job = $this->bqClient->runJob($loadConfig);`